sdks/python/apache_beam/ml/inference/pytorch_test.py

```diff
@@ -59,6 +63,23 @@ def forward(self, x):
     return out
 
 
+class PytorchLinearRegressionPredictionParams(torch.nn.Module):
+  def __init__(self, input_dim, output_dim):
+    super().__init__()
+    self.linear = torch.nn.Linear(input_dim, output_dim)
+
+  # k1 is the batched input, and prediction_param_array, prediction_param_bool
+  # are non-batchable inputs (typically model-related info) used to configure
+  # the model before its predict call is invoked
+  def forward(self, k1, k2, prediction_param_array, prediction_param_bool):
```

Comment:
why not just name the variables what they are instead of k1 and k2 and/or move the hash tag comments as doc for the method.